src/Workspaces/CSharp/Portable/Classification/Worker.cs

```diff
@@ -184,12 +184,12 @@ private void ClassifyTrivia(SyntaxTrivia trivia, SyntaxTriviaList triviaList)
                     return;
 
                 case SyntaxKind.SkippedTokensTrivia:
-                    ClassifySkippedTokens((SkippedTokensTriviaSyntax)trivia.GetStructure());
+                    ClassifySkippedTokens((SkippedTokensTriviaSyntax)trivia.GetStructure()!);
```

Comment:
How do we know this suppression is safe from surrounding code? Should this instead be an assertion?

Reply:
This is another really sutbtle one. Essentially we can't get `SyntaxKind.SkippedTokensTrivia` unless `trivia.UnderlyingNode` is a non-null value. This particular node is by definition a structured one hence it will always be non-null if there is a non-null `UnderlyingNode` value. 

I have no idea how to handle this though. There really isn't anything I can `Debug.Assert` here because it's a method return, not a property. I could add a new member to `StructuredTriviaSyntax` but that would be new public API surface. So far haven't added such a member for nullability yet (doesn't mean we shouldn't though).